cmd/podman/root.go

```diff
@@ -84,7 +85,15 @@ func init() {
 
 func Execute() {
 	if err := rootCmd.ExecuteContext(registry.GetContextWithOptions()); err != nil {
-		fmt.Fprintln(os.Stderr, "Error:", err.Error())
+		var message string
+		if errors.Cause(err) == define.ErrOCIRuntime {
+			// OCIRuntimeErrors include the reason for the failure in the
+			// second to last message in the error chain.
+			message = strings.TrimSuffix(err.Error(), ": "+define.ErrOCIRuntime.Error())
```

Comment:
Hm. Can we preface this with `define.ErrOCIRuntime.Error()` - so it's "Error: OCI Runtime error: writing file `cpu.cfs_quota_us`: Invalid argument"? Seems a little bit clearer

Reply:
Done